packages/odyssey-react/src/components/Select/Select/index.tsx

```diff
@@ -142,7 +142,7 @@ const Select = forwardRefWithStatics<HTMLSelectElement, Props, Statics>(
           {children}
         </select>
         <span className={styles.indicator} role="presentation">
-          <Caret />
+          <CaretDown />
```

Comment:
see my comment here about this naming convention for Icons: https://okta.slack.com/archives/G01JS0FE32P/p1631715796018600

I'd love to see us make these more clear by changing their names in our template to be `<IconCaretDown />` etc

Reply:
Im a +1 on this, was going to ask about doing this during standup today. I also have questions about the Icon component itself, but we can save that for OH!

Reply:
I prefer this prefixed style or similar. The visual design team has a much larger set of icons in the works, so it'd be nice to namespace these for clarity.